### PR TITLE
[TD-10828]<fix>: fix PyPI taospy connector failed for master branch

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: src/connector/python/README.md [hunk 1]
```diff
@@ -5,14 +5,16 @@
 
 ## Install
 
+You can use pip to install from github or pypi:
+
 ```sh
-git clone --depth 1 https://github.com/taosdata/TDengine.git
-pip install ./TDengine/src/connector/python
+pip install git+https://github.com/taosdata/taos-connector-python.git
+pip install taospy
```
Comment (sangshuduo) [inline src/connector/python/README.md:11]: pip isntall url
or
pip install taospy

?

Comment (zitsen) [reply src/connector/python/README.md:11]: `pip install url` not work for relative url for github projects, like  https://github.com/taosdata/TDengine/src/connector/python ,

Install with github project
```sh
pip install git+https://github.com/taosdata/taos-connector-python.git
pip install git@github.com:taosdata/taos-connector-python.git
```

Or install from pypi:

```sh
pip install taospy
```

Comment (zitsen) [reply src/connector/python/README.md:12]: `pip install url` not work for relative url for github projects, like  https://github.com/taosdata/TDengine/src/connector/python ,

Install with github project
```sh
pip install git+https://github.com/taosdata/taos-connector-python.git
pip install git@github.com:taosdata/taos-connector-python.git
```

Or install from pypi:

```sh
pip install taospy
```

Comment (sangshuduo) [reply src/connector/python/README.md:11]: I mean, we should use 'or' let the user know either way works well. Now it looks like we require users to execute two commands one by one.

**Review by sangshuduo — APPROVED** [review]

**Pull request merged**